content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -58,6 +58,12 @@ On the Console Settings page, there is a section for HTTP Proxy:
 
 ![PTFE HTTP Proxy Settings](./assets/ptfe-http-proxy.png)
 
+This change updates the proxy settings for the PTFE application services. To update the proxy settings for the installer (for example, to handle configuration tests correctly), additional steps are necessary:
+
+1. Locate the Replicated configuration files on the instance under either `/etc/sysconfig/` or `/etc/default`: `replicated` and `replicated-operator`.
+1. Open the files for editing. On the line that includes `REPLICATED_OPTS`, add `-e HTTP_PROXY=<your proxy url> -e NO_PROXY=<list of no_proxy hosts>` (see above for formatting of the `NO_PROXY` hosts) to the existing command options.
```

Comment:
Hmm, might as well just say it's a comma-separated list with no spaces, right?

Reply:
I'd maybe include a text box below with an example config line. (With multiple entries in the no_proxy section.)

Reply:
Hm, I'd rather not duplicate the config option here, since we have an example above, but I take Nick's point that maybe just specifying the formatting would be a good middle ground. @amy-hashi thoughts?

Reply:
Works for me.